Makefile.PL

```diff
@@ -486,9 +489,9 @@ sub c_o {
     my $inherited = shift->SUPER::c_o(@_);
 
     if ($^O =~ /Win32/ms && $Config{cc} eq 'cl') {  # Microsoft deprecated -o in favor of /Fo
-        $inherited =~ s{\$\*\.c\n}{/Fo\$\*.obj \$\*.c\n}msg;
+        $inherited =~ s{\$\*\.c[\s\t]*$}{/Fo\$\*.obj \$\*.c\n}msg;
```

Comment:
`\s` already include `\t`, so `[\s\t]*` can be simplified to `\s*`. Also I suppose real issue on Windows was not in `\t` but in `\r`, as usually, so `\r?\n` instead of `[\s\t]*$` will probably also works and will be much more clear.